action.yml

```
@@ -7,19 +7,41 @@ inputs:
   hax_repository:
     required: true
     default: "github:hacspec/hax"
+  fstar_reference:
+    required: false
+  fstar_repository:
+    required: true
+    default: "github:FStarLang/fstar"
 runs:
   using: "composite"
   steps:
     - uses: DeterminateSystems/nix-installer-action@main
+    - uses: DeterminateSystems/magic-nix-cache-action@main
```

`magic-nix-cache-action` will stop working in a week.

```suggestion
    - uses: DeterminateSystems/flakehub-cache-action@v7
```